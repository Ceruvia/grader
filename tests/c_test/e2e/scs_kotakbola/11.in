542 401 1000
M 30
M 16
B 82
M 30
M 56
M 77
M 82
M 65
B 61
H 3
M 60
E
