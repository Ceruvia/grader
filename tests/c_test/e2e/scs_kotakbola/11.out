3 416 143
Tidak
